drawBot/context/baseContext.py

```diff
@@ -620,6 +645,10 @@ def intersectionPoints(self, other=None):
             contours += other._contoursForBooleanOperations()
         return booleanOperations.getIntersections(contours)
 
+    def strokePath(self, width, lineCap="round", lineJoin="round", miterLimit=10):
+        strokedCGPath = Quartz.CGPathCreateCopyByStrokingPath(self._getCGPath(), None, width, _LINECAPSTYLESMAP[lineCap], _LINEJOINSTYLESMAP[lineJoin], miterLimit)
+        self._setCGpath(strokedCGPath)
```

Comment:
I think it should return fresh bezierPath object with a stroked path and not add it to itself. 
This is a similar action like the boolean operation actions

Reply:
Yeah sounds better!